clippy_dev/src/main.rs

```diff
@@ -75,10 +121,117 @@ fn main() {
                 update_lints(&UpdateMode::Change);
             }
         },
+        ("new_lint", Some(matches)) => {
+            create_new_lint(
```

Comment:
Like the `fmt` and `stderr_length_check`, can you put this subcommand in its own module?

Reply:
done